CGPs/cgp-0093/mainnet.json

```diff
@@ -0,0 +1,11 @@
+[
+  {
+    "contract": "GoldToken",
+    "function": "approve",
+    "args": [
+      "XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX",
```

Comment:
The multisig address must also be added here before merging

Reply:
@climateXcrypto thanks for updating the multisig in the readme. Could you please add it to the payload here aswell?